exercises/two-bucket/description.md

```diff
@@ -1,30 +1,34 @@
 # Description
 
-Given two buckets of different size, demonstrate how to measure an exact number of liters by strategically transferring liters of fluid between the buckets.
+Given two buckets of different size and which bucket to fill first, determine how many actions are required to measure an exact number of liters by strategically transferring fluid between the buckets. Since this mathematical problem is subject to a fair amount of interpretation and an individual's approach, the tests have been written specifically to expect one overarching solution.
```

Comment:
I don't really understand this statement "Since this mathematical problem is subject to a fair amount of interpretation and an individual's approach...". Yes, obviously, before we explain the rules, things are ambiguous, and yes, obviously, tests will expect specific answers. So I would like to hear your thoughts about getting rid of it. I don't think anything is lost.
 
```suggestion
Given two buckets of different size and which bucket to fill first, determine how many actions are required to measure an exact number of liters by strategically transferring fluid between the buckets.
```

Reply:
It was there before, my first draft attempt was to clarify but retain some of the original authors voice in the copy. I agree though, it is superfluous